src/clj/harja/kyselyt/urakat.sql

```diff
@@ -450,7 +454,13 @@ WHERE u.tyyppi = :urakkatyyppi :: urakkatyyppi
         exists(SELECT id
                FROM tekniset_laitteet_urakka tlu
                WHERE tlu.urakkanro = u.urakkanro AND
-                     st_dwithin(tlu.alue, st_makepoint(:x, :y), :threshold))))
+                     st_dwithin(tlu.alue, st_makepoint(:x, :y), :threshold)))
+       OR
+       ((:urakkatyyppi = 'siltakorjaus') AND
+        exists(SELECT id
+               FROM siltapalvelusopimus sps
+               WHERE sps.urakkanro = u.urakkanro AND
+                     st_dwithin(sps.alue, st_makepoint(:x, :y), :threshold))))
```

Comment:
Tässä siis siltakorjaus urakkatyyppi kohdalla ei hallintayksikkö tai sallitut urakat rajaus toimi?
foo AND bar OR baz => (foo AND bar) OR baz
AND on etusijalla, joten urakka palautetaan jos se on siltakorjaus ja riittävän lähellä, vaikka käyttäjällä ei olisi siihen oikeutta.